src/test/js-transform_test.ts

```diff
@@ -280,4 +280,28 @@ suite('jsTransform', () => {
     const result = jsTransform(input, {transformEsModulesToAmd: true});
     assert.equal(result.trim(), expected.trim());
   });
+
+  test('transforms import.meta', () => {
+    const rootDir =
+        path.join(__dirname, '..', '..', 'test-fixtures', 'npm-modules');
+    const filePath = path.join(rootDir, 'npm-module.js') as LocalFsPath;
```

Comment:
Don't actually need the `LocalFsPath` cast (optionally update the other cases too).

Reply:
done